owncloudData/src/main/java/com/owncloud/android/data/webfinger/OCWebfingerRepository.kt

```diff
@@ -0,0 +1,29 @@
+/**
+ * ownCloud Android client application
+ *
```

Comment:
You should add yourself as author in the new classes, we need someone to blame when this doesn't work 😀. Just joking, the merit is yours 🥇